src/pages/library/Showcase.js

```diff
@@ -16,104 +18,333 @@ import * as Style from 'src/libs/style'
-    ]),
     div({
       style: {
-        backgroundRepeat: 'no-repeat', backgroundPosition: 'center', backgroundSize: 'auto 100%', borderRadius: '0 5px 5px 0',
+        backgroundRepeat: 'no-repeat', backgroundPosition: 'center', backgroundSize: 'auto 100%', borderRadius: '5px 0 0 5px',
         width: 87,
         ...Utils.cond(
           [name.toLowerCase().includes('covid'), () => ({ backgroundImage: `url(${covidBg})` })],
           [variant === 'gatk', () => ({ backgroundColor: '#333', backgroundImage: `url(${gatkLogo})`, backgroundSize: undefined })],
           () => ({ backgroundImage: `url(${featuredBg})`, opacity: 0.75 })
         )
       }
-    })
+    }),
+    div({ style: { flex: 1, minWidth: 0, padding: '15px 20px', overflow: 'hidden' } }, [
+      div({ style: { display: 'flex' } }, [
+        div({ style: { flex: 1, color: colors.accent(), fontSize: 16, lineHeight: '20px', height: 40, marginBottom: 7 } }, [name]),
+        created && div([Utils.makeStandardDate(created)])
+      ]),
+      div({ style: { lineHeight: '20px', height: 100, whiteSpace: 'pre-wrap', overflow: 'hidden' } }, [description])
+      // h(MarkdownViewer, [description]) // TODO: should we render this as markdown?
+    ])
   ])
 }
 
+// collapsible section for sidebar categories
+const SidebarCollapser = ({ title, isOpened, onClick, children }) => {
+  return div({
+    role: 'group'
+  }, [
+    h(NavItem, {
+      onClick,
+      style: { ...styles.sidebarRow, ...styles.nav.navSection }
+    }, [
+      title,
+      div({ style: { flexGrow: 1 } }),
+      icon(isOpened ? 'angle-up' : 'angle-down', { size: 18, style: { flex: 'none' } })
+    ]),
+    div({
+      style: { flex: 'none' }
+    }, [h(RCollapse, { isOpened }, [children])])
+  ])
+}
+
+const groupByFeaturedTags = workspaces => _.flow([
+  _.map(tag => [tag, _.filter(w => _.includes(tag, w.tags.items), workspaces)]),
+  _.fromPairs
+])(uniqueSidebarTags)
+
+const Pill = ({ count, highlight }) => {
+  return div({
+    style: _.merge(styles.pill, highlight ? styles.pillHighlight : {})
+  }, [count])
+}
+
+const Sidebar = ({ onSectionFilter, onTagFilter, sections, selectedSections, selectedTags, workspacesByTag }) => {
+  const [collapsedSections, setCollapsedSections] = useState([])
+
+  const unionSectionWorkspaces = section => {
+    return _.uniq(_.flatMap(tag => workspacesByTag[tag], section.tags))
+  }
+
+  return div({ style: { display: 'flex', flexDirection: 'column' } }, [
+    _.map(section => {
+      return section.keepCollapsed ?
+        h(Clickable, {
+          key: section.name,
+          onClick: () => onSectionFilter(section),
+          style: { ...styles.sidebarRow, ...styles.nav.navSection }
+        }, [
+          div({ style: { flex: 1 } }, [section.name]),
+          h(Pill, {
+            count: _.size(unionSectionWorkspaces(section)),
+            highlight: _.includes(section, selectedSections)
+          })
+        ]) :
+        h(SidebarCollapser,
+          {
```

Comment:
```suggestion
        h(SidebarCollapser, {
```
(unless there is a reason for these choices that I am not seeing)